tutorials/Getting_Started/README.md

```diff
@@ -161,3 +161,9 @@ updates the UI.
 Running and debugging the `main()` function using run gutter is also supported.
 
 <img alt="Application running" src="screen6.png" height="500" />
+
+## Next steps
+
+Congratulations on getting your first Compose Multiplatform project working!  We encourage you to continue playing around with the areas that interest you, and look forward to seeing what you build!  When you're ready to continue learning, we have many more great tutorials available here: [Compose Multiplatform Tutorials](https://github.com/JetBrains/compose-jb/tree/master/tutorials).
```

Comment:
Thanks!

Better to change links to https://github.com/JetBrains/compose-jb#tutorials and https://github.com/JetBrains/compose-jb#examples, after we merge https://github.com/JetBrains/compose-jb/pull/2252

Reply:
I changed the links